ocd-3519.sql

```diff
@@ -0,0 +1,558 @@
+INSERT INTO openchpl.measure (measure_domain_id, required_test_abbr, measure_name, required_test, criteria_selection_required, removed, last_modified_user) SELECT (SELECT id FROM openchpl.measure_domain WHERE domain = 'EC'), 'RT15', 'Support Electronic Referral Loops by Receiving and Incorporating Health Information: Eligible Clinician', 'Required Test 15: Merit-based Incentive Payment System (MIPS) Promoting Interoperability Performance Category', false, false, -1;
+INSERT INTO openchpl.measure (measure_domain_id, required_test_abbr, measure_name, required_test, criteria_selection_required, removed, last_modified_user) SELECT (SELECT id FROM openchpl.measure_domain WHERE domain = 'EH/CAH Medicare PI'), 'RT15', 'Support Electronic Referral Loops by Receiving and Incorporating Health Information: Eligible Hospital/Critical Access Hospital', 'Required Test 15: Medicare Promoting Interoperability Program', false, false, -1;
+INSERT INTO openchpl.measure (measure_domain_id, required_test_abbr, measure_name, required_test, criteria_selection_required, removed, last_modified_user) SELECT (SELECT id FROM openchpl.measure_domain WHERE domain = 'EC'), 'RT2', 'Provide Patients Electronic Access to Their Health Information (formerly Patient Electronic Access): Eligible Clinician', 'Required Test 2: Merit-based Incentive Payment System (MIPS) Promoting Interoperability Performance Category', true, false, -1;
+INSERT INTO openchpl.measure (measure_domain_id, required_test_abbr, measure_name, required_test, criteria_selection_required, removed, last_modified_user) SELECT (SELECT id FROM openchpl.measure_domain WHERE domain = 'EC ACI Transition'), 'RT2', 'Patient Electronic Access: Eligible Clinician', 'Required Test 2: Promoting Interoperability Transition Objective 3 Measure 1', true, true, -1;
+INSERT INTO openchpl.measure (measure_domain_id, required_test_abbr, measure_name, required_test, criteria_selection_required, removed, last_modified_user) SELECT (SELECT id FROM openchpl.measure_domain WHERE domain = 'EC Group'), 'RT2', 'Patient Electronic Access: Eligible Clinician Group', 'Required Test 2: Stage 2 Objective 8 Measure 1 and Stage 3 Objective 5 Measure 1, ACI Transition Objective 3 Measure 1 and ACI Objective 3 Measure 1', true, false, -1;
+--INSERT INTO openchpl.measure (measure_domain_id, required_test_abbr, measure_name, required_test, criteria_selection_required, removed, last_modified_user) SELECT (SELECT id FROM openchpl.measure_domain WHERE domain = 'EH/CAH Medicare and Medicaid PI'), 'RT2', 'Provide Patients Electronic Access to Their Health Information (formerly Patient Electronic Access): Eligible Clinician', 'Required Test 2: Medicare and Medicaid Promoting Interoperability Programs', true, false, -1;
```

Comment:
Curious about these commented out ones

Reply:
Those were the measures that were duplicates, but had different names.  I have removed them from the script.